### ISSUE TITLE
certificate error

### ISSUE THREAD

**Issue by sandeepkumarmishra354** [WinkMeter/flutter_socket_io#1]
can't connect to server...
throwing exception 'connect error'
`D/FlutterSocketIoPlugin: SocketIO(28381): connect_error: [{"cause":{"cause":{"cause":{"certPath":{"mCertificates":[{"hash":-409637321,"type":"X.509"}],"type":"X.509"},"index":-1,"reason":"UNSPECIFIED","detailMessage":"Trust anchor for certification pathnot found.","stackTrace":[],"suppressedExceptions":[]},"detailMessage":"java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.","stackTrace":[],"suppressedExceptions":[]},"detailMessage":"java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.","stackTrace":[],"suppressedExceptions":[]},"detailMessage":"websocket error","stackTrace":[],"suppressedExceptions":[]}]
I/flutter (28381): [https://192.168.225.103:80/, FunctionId@60951892, FunctionId@60951892]
D/FlutterSocketIoPlugin: SocketIO(28381): reconnect_error: [{"cause":{"cause":{"cause":{"cause":{"certPath":{"mCertificates":[{"hash":-409637321,"type":"X.509"}],"type":"X.509"},"index":-1,"reason":"UNSPECIFIED","detailMessage":"Trust anchor for certification path not found.","stackTrace":[],"suppressedExceptions":[]},"detailMessage":"java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.","stackTrace":[],"suppressedExceptions":[]},"detailMessage":"java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.","stackTrace":[],"suppressedExceptions":[]},"detailMessage":"websocket error","stackTrace":[],"suppressedExceptions":[]},"detailMessage":"Connection error","stackTrace":[],"suppressedExceptions":[]}]`

**Comment by dinhha** [comment]
Are you running HTTPS on localhost? Looks your localhost have invalid SSL certificate

**Comment by sandeepkumarmishra354** [comment]
yes, i'm running HTTPS on local host with self signed certificate....
is there any way to accept self signed certificates

**Comment by LSmint** [comment]
@sandeepkumarmishra354 can you get solution on this?

**Comment by thealmamun** [comment]
Almost 2 years, still it's open, is there any solution?
